packages/ui/src/components/atoms/Checkbox.tsx

```diff
@@ -61,6 +61,11 @@ const CheckBoxLabelStyle = styled.label.attrs<ISize>(props => ({
     left: ${({ size }) => (size === 'big' ? '2px' : '1.5px')};
     background-image: ${({ theme }) => theme.palette.background.sideBar};
   }
+
+  ${CheckBoxStyle}:focus + & {
+    outline: 2px solid ${({ theme }) => theme.palette.background.sideBar};
+    outline: 2px solid #eef209;
```

Comment:
As mentioned line 67 works but 66, even the style doesnt get applied. Need your help on whether am I making any mistake with respect to template literal

Reply:
It won't work since `sideBar` is a gradient.

Reply:
background.sidebar is hex value, its there in color.styled
    sidebar: '#1E1A15',

Reply:
Changed the logic and handled it differently